dotCMS/src/integration-test/java/com/dotcms/translate/GoogleTranslationServiceIntegrationTest.java

```diff
@@ -0,0 +1,121 @@
+package com.dotcms.translate;
+
+import com.dotcms.datagen.SiteDataGen;
+import com.dotcms.datagen.TestDataUtils;
+import com.dotcms.datagen.TestUserUtils;
+import com.dotcms.security.apps.AppSecrets;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.liferay.portal.model.User;
+import com.liferay.util.StringPool;
+import com.dotmarketing.util.Config;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class GoogleTranslationServiceIntegrationTest {
+
+    private static User admin;
+    private static GoogleTranslationService translationService;
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5384201)